SWLOR.Game.Server/Feature/GuiDefinition/AreaManager.cs

```diff
@@ -0,0 +1,361 @@
+                .SetTitle("Area Manager")
+                .BindOnClosed(model => model.OnWindowClose())
+
+                .AddColumn(colSearch =>
+                {
+                    // Main Search - Top Window Row
+                    colSearch.AddRow(row =>
+                    {
+                        row.AddTextEdit()
+                            .SetPlaceholder("Search")
+                            .BindValue(model => model.SearchText);
+
+                        row.AddButton()
+                            .SetText("X")
+                            .SetHeight(35f)
+                            .SetWidth(35f)
+                            .BindOnClicked(model => model.OnClickClearSearch());
+
+                        row.AddButton()
+                            .SetText("Search")
+                            .SetHeight(35f)
+                            .BindOnClicked(model => model.OnClickSearch());
+                    });
+
+                    // Main Window Container - Second Window Row
+                    colSearch.AddRow(row =>
+                    {
+                        row.AddColumn(colAreas =>
+                        {
+                            colAreas.SetHeight(500f);
+                            colAreas.AddRow(row =>
+                            {
+                                row.AddList(template =>
+                                {
+                                    template.AddCell(cell =>
+                                    {
+                                        cell.AddToggleButton()
+                                            .BindText(model => model.AreaNames)
+                                            .BindIsToggled(model => model.AreaToggled)
+                                            .BindOnClicked(model => model.OnSelectArea());
+                                    });
+                                })
+                                    .BindRowCount(model => model.AreaNames);
+                            });
+
+                            colAreas.AddRow(row =>
+                            {
+                                row.AddSpacer();
+
+                                row.AddButton()
+                                    .SetText("Resave All Objects")
+                                    .SetHeight(35f)
+                                    .BindIsEnabled(model => model.IsResaveAllObjectsEnabled)
+                                    .BindOnClicked(model => model.OnClickResaveAllObjects());
+
+                                row.AddSpacer();
+                            });
+                        });
+
+                        row.AddColumn(colObjects =>
+                        {
+                            colObjects.SetHeight(450f);
+                            colObjects.SetWidth(325f);
+                            colObjects.AddRow(row =>
+                            {
+                                //row.SetHeight(450f);
```

Comment:
If this is intentionally commented out, let's go ahead and delete it to avoid confusion.